### ISSUE TITLE
Use INSERT INTO SELECT in technical solution

### ISSUE THREAD

**Issue by samay-sharma** [citusdata/citus_docs#198]
Currently, the technical solution uses a mix of custom functions and bash script to do INSERT INTO SELECT. Now that we have implemented INSERT INTO SELECT in 6.0, we should change our technical solution to use that.

**Comment by ozgune** [comment]
I'm also adding a note from @onderkalaci below.

>> (5) Three of our customers use HLL aggregation functions. Will Insert Into ... Select work with them? We determined the answer to that question to be yes.

We currently do not support stable and volatile functions on the queries. The simplest way to check the volatility of the functions is through pg_proc:
   SELECT proname, provolatile FROM pg_proc WHERE proname LIKE '%hll%' ORDER BY 2;

As far as I can see, all the related aggregate functions are immutable. Also, I tried to apply INSERT INTO ... SELECT to the hll examples in our docs@Real Time Dashboards. Here are the examples: https://gist.github.com/onderkalaci/6e8d93ae88cdbca3a11f64769441f4b2 

All looks fine with respect to hll related functions.

However, as I noted earlier, timestamptz related functions are stable. Thus, I changed timestampz -> timestamp to run the examples on the docs.
